suzieq/poller/worker/nodes/node.py

```diff
@@ -168,34 +148,30 @@ async def _init(self, **kwargs):
 
         self._init_service_queue()
 
-        if not self.port:
-            if self.transport == "ssh":
-                self.port = 22
-            elif self.transport == "https":
-                self.port = 443
-
         if self.transport == "ssh":
-            await self._init_ssh(init_boot_time=False)
+            self.port = self.port or 22
+            await self._init_ssh(init_dev_data=False)
+        elif self.transport == "https":
+            self.port = self.port or 443
```

Comment:
Why can't we try also here to send a message to a https device to check if it is reachable or not?

Reply:
Done.